src/lib.rs

```diff
@@ -95,3 +95,9 @@ pub type TLSServerTrustAnchors<'a> = TlsServerTrustAnchors<'a>;
 #[deprecated(note = "use TlsClientTrustAnchors")]
 #[allow(unknown_lints, clippy::upper_case_acronyms)]
 pub type TLSClientTrustAnchors<'a> = TlsClientTrustAnchors<'a>;
+
+// We don't operate on secret data so a convenient comparison function is warranted.
+#[must_use]
+fn equal(a: untrusted::Input, b: untrusted::Input) -> bool {
```

Comment:
WDYT about matching the name from https://github.com/rustls/webpki/commit/da923edaab56f599971e58773617fb574cd019dc ? In contrast I appreciate that it emphasizes the public data aspect of the change in the fn name.

Reply:
I don't really see the point in it because by intention all data this crate operates on is public, and also `==` and friends are an even bigger hazard. If I were to put support for non-public stuff into this project, I would split the verifier out into a separate crate.